mocks/menuItems.tsx

```diff
@@ -0,0 +1,93 @@
+import { ReactNode } from 'react';
+import Link from 'next/link';
+
+type PointMenu = {
+    label: ReactNode;
+    key: number;
+};
+type MenuItems = {
+    en: PointMenu[];
+    ka: PointMenu[];
+    ru: PointMenu[];
+};
+
+export const menuItems: MenuItems = {
+    en: [
+        {
+            label: (
+                <Link href="/home" passHref>
+                    Home
+                </Link>
+            ),
+            key: 1,
+        },
+        {
+            label: (
+                <Link href="/application" passHref>
+                    Application
+                </Link>
+            ),
+            key: 2,
+        },
+        {
+            label: (
+                <Link href="/contacts" passHref>
+                    Contacts
+                </Link>
+            ),
+            key: 3,
+        },
+    ],
```

Comment:
1) это не должно быть в папке mock потому что это не мок :)
2) ты добавил билиотеку для переводов поэтому тубу не нужно копипастить обхекты для трех локалей

```suggestion
        {
            label: (
                <Link href="/home" passHref>
                    {t('path to text Home')}
                </Link>
            ),
            key: 1,
        },
        {
            label: (
                <Link href="/application" passHref>
                       {t('path to text Application')}
                </Link>
            ),
            key: 2,
        },
        {
            label: (
                <Link href="/contacts" passHref>
                       {t('path to text Contacts')}
                </Link>
            ),
            key: 3,
        },
```